app/src/main/java/protect/card_locker/preferences/myAppCompatActivity.java

```diff
@@ -0,0 +1,58 @@
+package protect.card_locker.preferences;
```

Comment:
Can you please put the myAppCompatActivity class in the main location, instead of inside preferences?

Reply:
I would also prefer it if it was named something like `CatimaAppCompatActivity`

Reply:
I also see some unused imports (probably left over from testing), please run "Optimize Imports" in Android Studio if you can.

Reply:
all done:)